editor/script_create_dialog.cpp

```diff
@@ -656,14 +656,18 @@ void ScriptCreateDialog::_update_dialog() {
 		if (is_new_script_created) {
 			class_name->set_editable(true);
 			class_name->set_placeholder(TTR("Allowed: a-z, A-Z, 0-9, _ and ."));
-			class_name->set_placeholder_alpha(0.3);
+			Color placeholder_color = class_name->get_theme_color("font_placeholder_color");
+			placeholder_color.a = 0.3;
```

Comment:
It's not too important right now, as we have plenty of cases where this happens, but eventually everything like that should be defined as a custom property on the editor theme instead of hardcoding adjustments in place.